.github/workflows/scripts/parse-wp-version.js

```diff
@@ -0,0 +1,79 @@
+const https = require( 'http' );
+
+module.exports = async function parseWPVersion( wpVersion ) {
+    	// Start with versions we can infer immediately.
+	switch ( wpVersion ) {
+        case 'master':
```

Comment:
Confusing indentation here, but I assume that's the linter / formatter?

Reply:
VS Code did that :D Unfortunately our GitHub files don't actually have a linter. It's something I want to address but it isn't really a priority. All of our YAML files have different indentation styles too, as do various JSON files across the monorepo. There's some style-related stuff to look at.